src/app/components/connect-applications-wizard/connect-applications-wizard.html

```diff
@@ -0,0 +1,35 @@
+<div class="connect-apps-panel">
+  <!-- Header -->
+  <div class="connect-apps-wrap">
+    <div class="row">
+      <div class="col-md-12">
+        <table class="connect-apps-table table table-bordered">
```

Comment:
Not sure if you should use a table here. A table is used to display tabular data.